src/components/Banners/Banner.jsx

```diff
@@ -35,34 +55,57 @@ const Banner = () => {
                       Explore our brand new shop !
                     </button>
 
-                    <a href="https://github.com/IAmTamal/Milan" target="_blank" style={{ textDecoration: "none" }}>
+                    <a
+                      href="https://github.com/IAmTamal/Milan"
+                      target="_blank"
+                      style={{ textDecoration: "none" }}
+                    >
                       <button className="btn btn-warning banner_signup_btn banner_contribute_btn ">
                         <ImGithub className="banner_contribute_logo" />
-                        <p>
-                          Contribute to Milan
-                        </p>
+                        <p>Contribute to Milan</p>
                       </button>
                     </a>
                   </div>
                 ) : (
                   <div className="banner_btn_div">
                     <button
                       className="btn btn-warning banner_signup_btn"
-                      onClick={() => {
-                        nav("/clubs/register");
-                      }}
+                      onClick={handleSignupModalOpen}
                       data-cy="landingpage-club-signup"
                       id="landingpage-club-signup"
                     >
                       Sign up now
                     </button>
-
-                    <a href="https://github.com/IAmTamal/Milan" target="_blank" style={{ textDecoration: "none" }}>
+                    {signupModal && (
```

Comment:
- Can we have this in a separate file, maybe make a new folder inside `Banners` and then put the `.jsx` and `.css` within that folder ?